include/silo/common/alist.h

```diff
@@ -0,0 +1,26 @@
+#include <algorithm>
+#include <optional>
+#include <vector>
+
+template <typename K, typename V>
+class AList {
```

Comment:
I already mentioned that it was used, and it is now named AssociationList, and could be used again in some places.

Reply:
I would remove that class for now because I do not think it should use a raw reference there (implicitly deletes default constructor which should definitely be there..), and has no tests. This is also not quickly fixed